apps/sms/tools/update_nga_libs.sh

```diff
@@ -10,14 +10,14 @@
 # variables
 bower_path="bower_components"
 
-bridge_repo="gaia-components/bridge#master"
+bridge_repo="fxos-components/bridge#master"
```

Comment:
I don't do much changes here as probably we'll be migrating to NPM eventually, if I understood correctly :)